src/99-appendices/03-hed.md

```diff
@@ -1,63 +1,71 @@
-file to annotate events:  
+file to annotate events.
 
-Example:
+Example: An `*_events.tsv` annotated using HED tags for individual events.
 
 ```Text
-onset	duration	HED
-1.1	n/a	Event/Category/Experimental stimulus, Event/Label/CrossFix,  Sensory presentation/Visual, Item/Object/2D Shape/Cross
-1.3	n/a	Event/Category/Participant response, Event/Label/ButtonPress, Action/Button press
+onset  duration  HED
+1.1    n/a       Event/Category/Experimental stimulus, Event/Label/CrossFix,  Sensory presentation/Visual, Item/Object/2D Shape/Cross
+1.3    n/a       Event/Category/Participant response, Event/Label/ButtonPress, Action/Button press
 ...
 ```
 
 The direct approach requires that each line in the events file be
 annotated. Since there are typically thousands of events in each experiment,
-this method of annotation is usually not convenient unless the annotations are
-automatically generated. In many experiments, the event instances fall into a
-much smaller number of categories, and often these categories are labeled with
-numerical codes or short names. It is therefore more convenient to associate
-the HED annotations with these categories and allow the analysis tools to make
-the association with individual event instances during analysis. To use this
-approach, your `*_events.tsv` file should associate a category (often called an
-event code) with each event instance. Since BIDS allows an arbitrary number of
-columns to be included in an `*_events.tsv` file, you can make this association
-by including columns representing various types of event categories in your
-`*_events.tsv` file.
-
-Example:
+this method of annotation is not convenient unless the annotations are
+automatically generated. Usually annotations that appear in the `HED` column
+are specific to each individual event. Information that is common to groups
+of events can be annotated by category. Numerical values associated with each
+event can be annotated by value type.  Annotating by category and by value
+greatly reduces the effort required to HED tag data and improves the clarity
+for data users.
+
+## Annotating events by categories
+
+In many experiments, the event instances fall into a much smaller number of
+categories, and often these categories are labeled with numerical codes or short names.
+This categorical information usually corresponds to one or more columns in `*_events.tsv`
+representing categorical values. Instead of tagging this information for each
+individual event, you can assign HED tags for each distinct categorical value
+in an accompanying `*_events.json` sidecar and allow the analysis tools to make
+the association with individual event instances during analysis.  The column
+name in the `*_events.tsv` identifies the type of categorical variable. The following
+`*_events.tsv` file has one categorical variable called `mycodes` that takes
+on three possible values: `Fixation`, `Button`, and `Target`.
```

Comment:
line breaks:

```suggestion
In many experiments, the event instances fall into a much smaller number of
categories, and often these categories are labeled with numerical codes or short names.
This categorical information usually corresponds to one or more columns in `*_events.tsv`
representing categorical values.
Instead of tagging this information for each individual event,
you can assign HED tags for each distinct categorical value
in an accompanying `*_events.json` sidecar and allow the analysis tools to make
the association with individual event instances during analysis.
The column name in the `*_events.tsv` identifies the type of categorical variable.
The following `*_events.tsv` file has one categorical variable called `mycodes` that takes
on three possible values: `Fixation`, `Button`, and `Target`.
```